src/test/java/redis/clients/jedis/commands/jedis/BitCommandsTest.java

```diff
@@ -308,4 +309,286 @@ public void testBinaryBitfieldReadonly() {
     assertEquals(1L, responses2.get(0).longValue());
   }
 
+  @Test
+  @SinceRedisVersion("8.1.240")
+  public void bitOpDiff() {
+    // Use single-byte values for simplicity
+    byte[] key1 = new byte[]{(byte) 0x07}; // 00000111 - bits 0,1,2 set
+    byte[] key2 = new byte[]{(byte) 0x02}; // 00000010 - bit 1 set
+    byte[] key3 = new byte[]{(byte) 0x04}; // 00000100 - bit 2 set
```

Comment:
Using binary literals makes more sense :)

```suggestion
    byte[] key1 = new byte[]{(byte) 0b00000111}; // bits 0,1,2 set
    byte[] key2 = new byte[]{(byte) 0b00000010}; // bit 1 set
    byte[] key3 = new byte[]{(byte) 0b00000100}; // bit 2 set
```